cds_ils/migrator/xml_document_loader.py

```diff
@@ -86,6 +86,7 @@ def create_record(cls, dump):
         json_data = clean_created_by_field(json_data)
         add_cover_metadata(json_data)
         add_title_from_conference_info(json_data)
+        add_cds_url(json_data)
```

Comment:
warning here: this should be added only for NOT migrated records, otherwise CDS will redirect to ILS.